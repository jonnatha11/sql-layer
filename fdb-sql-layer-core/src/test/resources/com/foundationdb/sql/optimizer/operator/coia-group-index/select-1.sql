SELECT customers.name,order_date,sku,quan
FROM customers,orders,items
WHERE customers.cid = orders.cid
AND orders.oid = items.oid
AND customers.name = 'Smith'
AND items.sku = '1234'