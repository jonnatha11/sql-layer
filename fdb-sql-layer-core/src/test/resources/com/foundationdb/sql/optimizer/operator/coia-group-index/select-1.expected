PhysicalSelect[name:varchar(32), order_date:date, sku:varchar(32), quan:int]
  Project_Default(customers.name, orders.order_date, items.sku, items.quan)
    Flatten_HKeyOrdered(customers - orders INNER items)
      Flatten_HKeyOrdered(customers INNER orders)
        GroupLookup_Default(Index(customers.cname_and_sku) -> customers, orders, items)
          IndexScan_Default(Index(customers.cname_and_sku), customers.name = 'Smith', items.sku = '1234')
